lib/reporters/verbose.js

```diff
@@ -24,29 +24,32 @@ class VerboseReporter {
 		return '';
 	}
 	test(test, runStatus) {
+		const lines = [];
 		if (test.error) {
-			return '  ' + colors.error(figures.cross) + ' ' + test.title + ' ' + colors.error(test.error.message);
```

Comment:
This whole stream of changes looks more noisy than it is. This is simply switching from an immediate `return` approach to appending to an output array so that we can concat comments after the test summary.